### ISSUE TITLE
Improve label's steps

### ISSUE THREAD

**Issue by Dalanir** [ToucanToco/vue-query-builder#288]
Here will lie our propositions for them...

For now, there it is:

- [ ] aggregate: `Aggregate on ${columnNames joined by pipes}`

- [ ] argmax: `max value on column ${columnName}"

- [ ] argmin: `min value on column "${columnName}"`

- [ ] custom: 'custom step'

- [ ] domain: `use domain ${domainName}`

- [ ] duplicate: `duplicate column "${columnName}" in "${newColumnName}"`

- [ ] delete: `delete columns "${columnNames joined by pipes}`;

- [ ] fillna: `replace empty values of "${columnName}" with "${value}"`

- [ ] formula: `${formula}`

- [ ] percentage: `% of ${columnName}`;

- [ ] pivot: `pivot ${columnToPivotName} on ${indexColumns joined by commas}`;

- [ ] rename:  `rename column ${columnOldName} into ${columnNewName}`;

- [ ] replace: `replace value ${valueToReplace} with ${newValue} in column ${columnName}`;

- [ ] select: `select columns (${columns joined by commas})`;

- [ ] sort: `sort columns ${sign that represent the order}`

- [ ] top: `top values on ${columnName}`

- [ ] unpivot: `unpivot on ${columnNames joined by commas}`

- [ ] filter:

- eq: `${condition.column} = ${condition.value}`;
- ne: `${condition.column} ≠ ${condition.value}`;
- gt: `${condition.column} > ${condition.value}`;
- ge: `${condition.column} >= ${condition.value}`;
- lt: `${condition.column} < ${condition.value}`;
- le: `${condition.column} <= ${condition.value}`;
- in: `${condition.column} in (${condition.value.join(', ')})`;
- nin: `${condition.column} not in (${condition.value.join(', ')})`;

**Comment by vdestraitt** [comment]
General comments:

- I think we should begin every label by an uppercase character.

- When we say "columnNames joined by pipes", do we mean joined by the pipe character "|" ? If so, I think it is a bit geeky ;),  and that we should rather use commas.

- May a label become too long, we can crop it with trailing "...", and display the full label on hover

My proposal:

-  aggregate: `Aggregate ${value1, value2} grouped by ${dimension1, dimension2}`


- argmax/argmin: `Keep row with maximum/minimum in column ${valueCol}`

- fillna: `Replace null values in ${columnName} by ${value}`

- formula: `Compute ${value1 + value2 / 2} in column ${column}`

- percentage: `Compute the row-level percentage of total in ${valueCol}`

-  pivot: `pivot column ${columnToPivotName}`
(I think it is enough here, it is too wordy to precise which columns to keep fixed, what is import is which column we reshape)

- replace: `Replace ${valueToReplace} with ${newValue} in column ${columnName}`
or when we have several values to replaces, simply: `Replace values in column ${columnName}`

- select: `Keep columns ${column1, column2, column3}`

- sort: `Sort columns ${column1, column2}`
I think that it is enough here to just specify the columns impacted without giving the detail of sorting order

- top: `Get top ${10} values in column ${columnName}`

- unpivot: `Unpivot columns ${column1, column2}`
Columns to e specified are in the `unpivot` parameter of the step
